.gitignore

```diff
@@ -8,3 +8,4 @@
 
 # kept because of historical reasons
 /npi_token
+.idea/
```

Comment:
Please put specific ignore config for you into your global gitignore

https://git-scm.com/docs/gitignore